### ISSUE TITLE
Activate homepage rotating gallery on device independent control

### ISSUE THREAD

**Issue by caseyedavis12** [WGBH/AAPB2#1079]
The rotating gallery on the homepage should be activated using device independent control.

**Comment by sroosa** [comment]
@jasoncorum was this resolved? should we close the ticket?
